.github/workflows/workflow.yml

```diff
@@ -45,6 +42,23 @@ jobs:
       run: tox -e copyright-check
     - name: Manifest check
       run: tox -e check-manifest
```

Comment:
Should we consider running these in parallel?

Reply:
Also re-enable pylint?

Reply:
it's code check, dont think it's matter where we put this tasks